### DISCUSSION TITLE
Caching with wire:navigate.hover

### DISCUSSION THREAD

**Discussion by yamakrane** [livewire/livewire#9006] [1. Help]
We have a reactive web app and we like a lot using wire:navigate and more especially wire:navigate.hover. But I realized that navigating between pages using wire:navigate.hover caches pages somehow instead of only prefetching the page on hover. We have many reactive components and navigating.hover between those pages ruins it. So I want it know if there is a way to disable caching on wire:navigate.hover for some specific pages or for all the pages if not. Thanks.

**Comment by danie-ramdhani** [comment]
Try disable it in livewire:navigate event listener.

https://livewire.laravel.com/docs/navigate#javascript-hooks

**Reply by yamakrane** [reply]
Yeah, but how, as I don't even know how Livewire is caching pages when using wire:navigate.hover  😢

**Comment by joshhanley** [comment]
@yamakrane thanks for reporting! We're currently in the process of investigating all navigate issues and we'd like to know a bit more about what you would expect to happen here.

We recently change it so if you use `.hover`, it will cache the page for 30 seconds. What you are asking is to disable caching for specific pages, but that defeats the purpose of `.hover` in my opinion. We need to store the resolved page for a period of time so that way when the click does happen, it doesn't make another request. But if the user hovers over a link for too long, then we would also want a new request to be made at that point, as there might be stale data.

So keeping the above in mind, what exactly are you after and how are you hoping it will work?

**Reply by kachelle** [reply]
Josh! It would be lovely to be able to set for example the cached duration on for example 5 seconds. (as mentioned here: https://github.com/livewire/livewire/pull/8175#issuecomment-3406922483)

For more reactive apps, that if you by chance hovered over a link, but didn't click. Then added something to the page, so it reflects the newly added things on the page you hovered before, and now clicked on.
